starter_code/src/movies.js

```diff
@@ -1,20 +1,152 @@
-/* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(array) {
+  return array.map(function(movie){
+    var newMovie = Object.assign({},movie);
+    var split = newMovie.duration.split(' ');
+    var hours;
+    var minutes;
+    if (split.length > 1){
+      hours = Number(split[0].replace('h', ''));
+      minutes = Number(split[1].replace('min', ''));
+    } else {
+      if (split[0].indexOf('h') > -1) {
+        hours = Number(split[0].replace('h', ''));
+        minutes = 0;
+      } else {
+        hours = 0;
+        minutes = Number(split[0].replace('min', ''));
+      }
+    }
+  newMovie.duration = hours*60 + minutes;
+  return newMovie;
+  });
```

Comment:
Demasiado largo, mejor dividid en varias funciones con tareas específicas. Además, podéis solucionarlo con la mitad de `replace()`